jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -36,6 +36,12 @@ function prepareEnv() {
     unset KIE_SERVER_KAFKA_EXT_GROUP_ID
     unset KIE_SERVER_KAFKA_EXT_MAX_BLOCK_MS
     unset KIE_SERVER_KAFKA_EXT_TOPICS
+    unset KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_ACKS
```

Comment:
ANd, please, document this envs on the image.yaml descriptor for the kieserver image.

Reply:
https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/551